DataRepo/utils/accucor_data_loader.py

```
@@ -705,8 +754,15 @@ def load_data(self):
                         # cache
                         inserted_peak_group_dict[peak_group_name] = peak_group
 
+                except ValidationError as ve:
+                    if self.is_a_downstream_dupe_error(ve, index, "corrected"):
+                        # This is a redundant error caused by the existence of a DupeCompoundIsotopeCombos, so
+                        # we can ignore this error
+                        pass
```

C. If the cause of this error is related to a previous duplicate isotopes error, this error is ignored, because it will go away when the dupes are addressed.